src/docfx/docfx.csproj

```diff
@@ -22,6 +22,8 @@
     <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="2.0.298" />
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.13.1" />
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="3.1.3" />
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.4" />
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.3.1" />
```

Comment:
OK... is that any best practice document about this? since in the official Microsoft Keyvault access samples, it is still using `Microsoft.Azure.Services.AppAuthentication`

Reply:
https://github.com/Azure/azure-sdk-for-net#client

Reply:
There is a new wave of .NET SDK to access azure. KeyVault is using `Azure.Security.KeyVault.*`, samples are typically not updated as frequent.